stdlib/bir/src/main/ballerina/bir/bir_func_body_parser.bal

```diff
@@ -87,6 +90,12 @@ public type FuncBodyParser object {
             var lhsOp = self.parseVarRef();
             NewMap newMap = {kind:kind, lhsOp:lhsOp, typeValue:bType};
             return newMap;
+        } else if (kindTag == INS_NEW_INST) {
+            var defIndex = self.reader.readInt32();
+            kind = INS_KIND_NEW_INST;
+            var lhsOp = self.parseVarRef();
+            NewInstance newBhv = {kind:kind, lhsOp:lhsOp, typeDef: self.findTypeDef(defIndex)};
+            return newBhv;
```

Comment:
Bhv? does it stand for anything :)

Reply:
Before, I had an object called `Behaviors` here, will change.